src/webkit.cs

```diff
@@ -708,23 +708,59 @@ partial interface DomDocument {
 		[Export ("cookie", ArgumentSemantic.Copy)]
 		string Cookie { get; set;  }
 
-		[Export ("body", ArgumentSemantic.Retain)]
+#if !XAMCORE_5_0
+		[Obsolete ("Use the 'Body' property instead.")]
+		[Wrap ("Body", IsVirtual = true)]
```

Comment:
Why do these Wraps have the 'IsVirtual = true'? Are they being overridden in the generator?

Reply:
I had the same question when I began fixing these bugs. Adding the virtual keyword ensures that the api has no breaking changes when these capitalization issues are fixed.